### ISSUE TITLE
Long list of commands get hidden and cannot edit at top row in preview mode

### ISSUE THREAD

**Issue by Truncated** [mugiwara85/CodeblockCustomizer#95]
The top line in preview mode doesn't expand to size 2+ lines when wrapping kicks in.

Preview mode on top with Obsidian theme and no custom CSS enabled
Source mode on bottom, same conditions

![image](https://github.com/mugiwara85/CodeblockCustomizer/assets/14208325/4fea6ba9-a2dc-4921-b13c-147a20341e26)
![image](https://github.com/mugiwara85/CodeblockCustomizer/assets/14208325/8f8d1b29-fe3c-49d3-975d-68b303b90618)

<details><summary>Example to test:</summary>
<p>

```
```cwf hl:1|/:er,2-14|{:},4|De:ow,6|:ers,7|:es,10|:ations,11|:ions,12|:ls,13|:ing
// ! SECTION: 00 Workflow Macro Wrapper
{workflow:name=Workflow Name In Setting|key=spaceworkflow-234576211|label=label-used-to-apply|adminusers=@workflowAdmins@,@wf-SomeWorkflowRole@|stickylabels=label-used-to-apply,another_label,subpagelabel|content=pages}
    {description}
    Details about this workflow
    {description}
// ! SECTION: 02 Workflow Parameters
// ! SECTION: 03 States
	{state=Start}
	{state}
// ! SECTION: 04 Workflow Pseudo-Functions and Calculations
// ! SECTION: 05 Workflow Steps and Permissions
// ! SECTION: 06 Instruction Messages and Emails
// ! SECTION: 99 Error handling
{workflow}

```

</p>
</details>

**Comment by mugiwara85** [comment]
I didn't even noticed this. I will look into it

**Comment by mugiwara85** [comment]
@Truncated This should also be ok now.

**Comment by mugiwara85** [comment]
Fixed in version 1.2.8
